### PR TITLE
Groom changelog /1.3.0

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

groom changelog
# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: GROOMME.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Please groom the changelog and then delete me.
```
Comment (JamesBurnside) [inline GROOMME.md:1]: delete ?

File: packages/communication-react/CHANGELOG.md [hunk 2]
```diff
@@ -6,23 +6,21 @@ This log was last generated on Mon, 13 Jun 2022 18:29:25 GMT and should not be m
 
 ## [1.3.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.3.0)
 
+This stable release is mostly minor changes and bug fixes, there are no major feature introductions.
+
+A lot of bug fixes in this stable release are related to accessibility. Fixes include improved narration from screen readers in CallWithChat composite as well as improved keyboard navigation.
+
+Included as well are updates to the video streams with new features like a loading spinner in the remote stream when the remote participants connection is loading. As well we introduced the ability to update the local video streams scaling mode without destroying the current local video stream.
```
Comment (JamesBurnside) [inline packages/communication-react/CHANGELOG.md:13]: This is just a beta feature for composites, but stable for components --- so maybe best not to mention the loading spinner here

File: packages/communication-react/CHANGELOG.md [hunk 3]
```diff
@@ -6,23 +6,21 @@ This log was last generated on Mon, 13 Jun 2022 18:29:25 GMT and should not be m
 
 ## [1.3.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.3.0)
 
+This stable release is mostly minor changes and bug fixes, there are no major feature introductions.
```
Comment (JamesBurnside) [inline packages/communication-react/CHANGELOG.md:9]: nit/suggestion: a common, nice addition when nothing major has change is "quality of life improvements"
```suggestion
This stable release contains minor changes, bug fixes and quality of life improvements, there are no major feature introductions.
```

File: packages/communication-react/CHANGELOG.md [hunk 4]
```diff
@@ -6,23 +6,21 @@ This log was last generated on Mon, 13 Jun 2022 18:29:25 GMT and should not be m
...
+
+Included as well are updates to the video streams with new features like a loading spinner in the remote stream when the remote participants connection is loading. As well we introduced the ability to update the local video streams scaling mode without destroying the current local video stream.
+
 Mon, 13 Jun 2022 18:29:25 GMT 
 [Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.2.2-beta.1...@azure/communication-react_v1.3.0)
 
 ### Minor changes
-
-- `@internal/react-components`
   - Show a loading spinner when remote video stream is loading ([PR #1954](https://github.com/azure/communication-ui-library/pull/1954) by chwhilar@microsoft.com)
```
Comment (JamesBurnside) [inline packages/communication-react/CHANGELOG.md:19]: would update this to 'Video Gallery supports a loading spinner to indicate a video stream is incoming but has not yet loaded. This feature will be enabled in the Composites soon.'

File: packages/communication-react/CHANGELOG.md [hunk 5]
```diff
@@ -57,77 +52,27 @@ Mon, 13 Jun 2022 18:29:25 GMT
   - updating beta @azure/communication-calling to 1.5.4-beta.1 ([PR #1925](https://github.com/azure/communication-ui-library/pull/1925) by 79475487+mgamis-msft@users.noreply.github.com)
   - Change Microphone Contextual Menu header text to Audio Device when no speakers are detected ([PR #1883](https://github.com/azure/communication-ui-library/pull/1883) by edwardlee@microsoft.com)
   - Remove spurius console log ([PR #1451](https://github.com/azure/communication-ui-library/pull/1451) by 82062616+prprabhu-ms@users.noreply.github.com)
-- `@internal/storybook`
   - updating beta @azure/communication-calling to 1.5.4-beta.1 ([PR #1925](https://github.com/azure/communication-ui-library/pull/1925) by 79475487+mgamis-msft@users.noreply.github.com)
```
Comment (JamesBurnside) [inline packages/communication-react/CHANGELOG.md:55]: remove `updating beta @azure/communication-calling to 1.5.4-beta.1`

File: packages/communication-react/CHANGELOG.md [hunk 6]
```diff
@@ -57,77 +52,27 @@ Mon, 13 Jun 2022 18:29:25 GMT
...
   - Resolve icon centering on SendBox storybook examples ([PR #1833](https://github.com/azure/communication-ui-library/pull/1833) by edwardlee@microsoft.com)
   - patched error on storybook participant item not showing name ([PR #1844](https://github.com/azure/communication-ui-library/pull/1844) by carolinecao@microsoft.com)
   - Add Best Practices documentation to Storybook ([PR #1970](https://github.com/azure/communication-ui-library/pull/1970) by 2684369+JamesBurnside@users.noreply.github.com)
-- `@internal/acs-ui-common`
   - Add default replacer to do safe stringify ([PR #1921](https://github.com/azure/communication-ui-library/pull/1921) by jinan@microsoft.com)
-- `@internal/calling-component-bindings`
   - Update VideoGallery bindings to return the created view when starting a local video stream ([PR #1891](https://github.com/azure/communication-ui-library/pull/1891) by 2684369+JamesBurnside@users.noreply.github.com)
   - updating beta @azure/communication-calling to 1.5.4-beta.1 ([PR #1925](https://github.com/azure/communication-ui-library/pull/1925) by 79475487+mgamis-msft@users.noreply.github.com)
```
Comment (JamesBurnside) [inline packages/communication-react/CHANGELOG.md:63]: remove `updating beta @azure/communication-calling to 1.5.4-beta.1`

File: packages/communication-react/CHANGELOG.md [hunk 7]
```diff
@@ -57,77 +52,27 @@ Mon, 13 Jun 2022 18:29:25 GMT
...
+  - Add callIdHistory to context + internal context - Update all visit to callId using latestCallId - Encapsule the access to internalMap directly to ensure no leaks ([PR #1817](https://github.com/azure/communication-ui-library/pull/1817) by jiangnanhello@live.com)
   - Update the right call.id into state ([PR #1818](https://github.com/azure/communication-ui-library/pull/1818) by jiangnanhello@live.com)
-- `@internal/chat-component-bindings`
   - Remove dependency on azure/communication-signalling, instead use types from azure/communication-chat ([PR #1895](https://github.com/azure/communication-ui-library/pull/1895) by 2684369+JamesBurnside@users.noreply.github.com)
-- `@internal/chat-stateful-client`
   - Remove dependency on azure/communication-signalling, instead use types from azure/communication-chat ([PR #1895](https://github.com/azure/communication-ui-library/pull/1895) by 2684369+JamesBurnside@users.noreply.github.com)
```
Comment (JamesBurnside) [inline packages/communication-react/CHANGELOG.md:72]: duplicate entry for `Remove dependency on azure/communication-signalling`

File: packages/communication-react/CHANGELOG.md [hunk 8]
```diff
@@ -6,23 +6,21 @@ This log was last generated on Mon, 13 Jun 2022 18:29:25 GMT and should not be m
...
-  - update dominantspeaker selector to work with videoGalleryUtils function changes. ([PR #1929](https://github.com/azure/communication-ui-library/pull/1929) by 94866715+dmceachernmsft@users.noreply.github.com)
-- `@internal/calling-component-bindings`
-  - Update the return from the `ParticipantList` selector to handle PSTN users. ([PR #1929](https://github.com/azure/communication-ui-library/pull/1929) by 94866715+dmceachernmsft@users.noreply.github.com)
 
 ### Patches
-
-- `@internal/react-components`
   - Fix support for onDisposeLocalStreamView in `VideoGallery` ([PR #1866](https://github.com/azure/communication-ui-library/pull/1866) by 2684369+JamesBurnside@users.noreply.github.com)
   - Fixed theming of links for html and richtext/html messages in MessageThread. ([PR #1824](https://github.com/azure/communication-ui-library/pull/1824) by miguelgamis@microsoft.com)
   - Added error text color in light and dark themes in semanticColors property. ([PR #1861](https://github.com/azure/communication-ui-library/pull/1861) by 79475487+mgamis-msft@users.noreply.github.com)
```
Comment (JamesBurnside) [inline packages/communication-react/CHANGELOG.md:26]: L27 below this: `Revert directly calling updateScalingMode in the VideoGallery VideoTiles` - remove this and line this PR reverts

File: packages/communication-react/CHANGELOG.md [hunk 9]
```diff
@@ -57,77 +52,27 @@ Mon, 13 Jun 2022 18:29:25 GMT
...
-  - add placeholders for PSTN functions in the adapters for when time to add to composites ([PR #1914](https://github.com/azure/communication-ui-library/pull/1914) by 94866715+dmceachernmsft@users.noreply.github.com)
-  - Added Icon button for upload button, cancel file upload icon and download icon to make it tab navigable ([PR #1881](https://github.com/azure/communication-ui-library/pull/1881) by 97124699+prabhjot-msft@users.noreply.github.com)
-  - Update the Call and CallWithChat Adapters to use `onToggleHold` and `onAddParticipant` calling handlers ([PR #1973](https://github.com/azure/communication-ui-library/pull/1973) by 94866715+dmceachernmsft@users.noreply.github.com)
-  - Performance improvement by preventing CallWithChat composite reloads when toggling side panes on mobile devices ([PR #1941](https://github.com/azure/communication-ui-library/pull/1941) by anjulgarg@live.com)
-  - Updating @azure/communication-chat to 1.2.0 ([PR #1815](https://github.com/azure/communication-ui-library/pull/1815) by anjulgarg@live.com)
```
Comment (JamesBurnside) [inline packages/communication-react/CHANGELOG.md:105]: This looks like a stable change that should be included?

File: packages/communication-react/CHANGELOG.md [hunk 10]
```diff
@@ -57,77 +52,27 @@ Mon, 13 Jun 2022 18:29:25 GMT
...
-- `@internal/react-composites`
-  - Rework how Custom buttons are injected in Call Composite and add Custom Button Injection into CallWithChat Composite ([PR #1931](https://github.com/azure/communication-ui-library/pull/1931) by edwardlee@microsoft.com)
-  - add placeholders for PSTN functions in the adapters for when time to add to composites ([PR #1914](https://github.com/azure/communication-ui-library/pull/1914) by 94866715+dmceachernmsft@users.noreply.github.com)
-  - Added Icon button for upload button, cancel file upload icon and download icon to make it tab navigable ([PR #1881](https://github.com/azure/communication-ui-library/pull/1881) by 97124699+prabhjot-msft@users.noreply.github.com)
-  - Update the Call and CallWithChat Adapters to use `onToggleHold` and `onAddParticipant` calling handlers ([PR #1973](https://github.com/azure/communication-ui-library/pull/1973) by 94866715+dmceachernmsft@users.noreply.github.com)
-  - Performance improvement by preventing CallWithChat composite reloads when toggling side panes on mobile devices ([PR #1941](https://github.com/azure/communication-ui-library/pull/1941) by anjulgarg@live.com)
```
Comment (JamesBurnside) [inline packages/communication-react/CHANGELOG.md:104]: `Performance improvement by preventing CallWithChat composite reloads when toggling side panes on mobile devices` looks like a stable change

**Review by JamesBurnside — APPROVED** [review]
Left some suggestions but looks great!

**Review by prprabhu-ms — APPROVED** [review]

**Pull request merged**